smoke-tests-otel-starter/spring-boot-reactive-common/src/main/java/io/opentelemetry/spring/smoketest/AbstractOtelReactiveSpringStarterSmokeTest.java

```diff
@@ -68,16 +68,13 @@ void webClientAndWebFluxAndR2dbc() {
                                 assertThat(s.getName())
                                     .isEqualToIgnoringCase("SELECT testdb.PLAYER"))
                         .hasAttribute(DbIncubatingAttributes.DB_NAME, "testdb")
-                        .hasAttributesSatisfying(
-                            a ->
-                                assertThat(a.get(DbIncubatingAttributes.DB_SQL_TABLE))
-                                    .isEqualToIgnoringCase("PLAYER"))
-                        .hasAttribute(DbIncubatingAttributes.DB_OPERATION, "SELECT")
+                        // 1 is not replaced by ?,
+                        // otel.instrumentation.common.db-statement-sanitizer.enabled=false
```

Comment:
(btw, I don't think it should be replaced even with sanitization enabled, I opened #11379)